exercises/hangman/hangman_test.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+import hangman
+
+
+class HangmanTest(unittest.TestCase):
+    def test(self):
+        self.assertEqual("hangman", "hangman")
+
```

Comment:
More tests should be added

Reply:
@vaibhavsingh97, I would suggest basing the tests on the ones from the [C# track](https://github.com/exercism/csharp/blob/master/exercises/hangman/HangmanTest.cs) since they have fairly good test coverage.